src/main/java/org/gitlab/api/GitlabAPI.java

```diff
@@ -935,6 +935,32 @@ public GitlabProject createUserProject(Integer userId, String name, String descr
     }
 
     /**
+     *
+     * @param namespace The namespace of the fork
+     * @param projectId ProjectId of the project forked
+     * @return The new Gitlab Project
+     * @throws IOException on gitlab api call error
+     */
+    public GitlabProject createFork(String namespace,Integer projectId) throws IOException{
+        Query query = new Query()
+                .appendIf("id", projectId)
+                .append("namespace", namespace);
+        String tailUrl = GitlabProject.URL+"/" + projectId + "/fork";
+        return dispatch().to(tailUrl,GitlabProject.class);
```

Comment:
nit: space after comma

Reply:
There are a few more of these elsewhere in the PR; these are just the first two I noticed.

Reply:
I fix it with my ide settings